packages/schemas/alterations/1.0.0_beta.10-1-logto-config.ts

```diff
@@ -1,6 +1,6 @@
 import { sql } from 'slonik';
 
-import type { AlterationScript } from '../src/types/alteration.js';
+import type { AlterationScript } from '../lib/types/alteration.js';
```

Comment:
@wangsijie i found the `src` import will cause a different output file structure and lead to no alteration available. so i reverted this